easybuild/easyconfigs/n/ncurses/ncurses-6.0.eb

```diff
@@ -14,6 +14,9 @@ toolchainopts = {'optarch': True, 'pic': True}
 source_urls = [GNU_SOURCE]
 sources = [SOURCE_TAR_GZ]
 
+# ncurses-6.0 builds fail with `mawk`, so force use of `gawk`
+# see http://lists.gnu.org/archive/html/bug-ncurses/2015-08/msg00013.html
+preconfigopts = "env AWK=/usr/bin/gawk "
```

Comment:
I can change that to read `AWK=/usr/bin/awk` but then it relies on admins to do `update-alternatives --set awk /usr/bin/gawk` e.g. it might not work out of the box on Debian ...